style.css

```diff
@@ -468,3 +474,269 @@ form > * {
 form textarea {
   height: 17vh;
 }
+
+.contact-section p {
+  color: white;
+}
+
+@media only screen and (min-width: 992px) {
+  .header {
+    display: flex;
+    flex-direction: row;
+    padding-top: 2%;
+    padding-left: 6%;
+    width: auto;
+  }
+
+  #hamb-icon {
+    display: none;
+  }
+
+  .nav-cmp {
+    display: flex;
+    flex-direction: row;
+    text-decoration: none;
+    justify-content: space-between;
+    width: 30%;
+    padding-left: 7%;
+    margin-right: 9%;
+  }
+
+  .wrapper {
+    display: flex;
+    flex-direction: column;
+    max-width: 100%;
+    background: url(img/big-screen-bg.png);
+    background-repeat: no-repeat;
+    background-size: cover;
+    background-color: #fff;
+    height: 105vh;
+    border-bottom-left-radius: 13rem;
+  }
+
+  .container {
+    margin-left: 10%;
+    flex-wrap: wrap;
+    margin-top: 17%;
+  }
+
+  .container h1 {
+    font-size: 48px;
+    line-height: 60px;
+    align-items: center;
+    letter-spacing: 0.37px;
+    margin-top: 0%;
+    margin-bottom: 2%;
+  }
+
+  .container #title-break {
+    display: flex;
+  }
+
+  .container .about-me {
+    margin-right: 25%;
+    font-style: normal;
+    font-weight: 300;
+    font-size: 20px;
+    line-height: 28px;
+    margin-bottom: 2%;
+  }
+
+  .social-ic {
+    margin-bottom: 10%;
+    width: 15%;
+  }
+
+  .nav-itm {
+    user-select: none;
+    color: #344563;
+    text-decoration: none;
+  }
+
+  .nav-itm:hover {
+    text-decoration: underline;
+  }
+
+  .card-work {
+    margin-bottom: 10%;
+  }
+
+  #work1,
+  #work3 {
+    display: flex;
+    flex-direction: row;
+    padding: 30px;
+  }
+
+  #work2,
+  #work4 {
+    display: flex;
+    flex-direction: row-reverse;
+    padding: 30px;
+  }
+
+  #work2 .work-details,
+  #work4 .work-details {
+    display: flex;
+    margin-right: 60px;
+  }
+
+  .card-work img {
+    height: 55vh;
+    display: flex;
+  }
+
+  #work1 .work-details,
+  #work3 .work-details {
+    display: flex;
+    margin-left: 60px;
+  }
+
+  .work-details a {
+    font-size: 17px;
+    width: 35%;
+    margin-top: 30px;
+    padding: 10px;
+  }
+
+  .work-details a:hover {
+    background-color: #4053fc;
+    color: white;
+  }
+
+  .work-details h1 {
+    margin-top: 30px;
+    font-size: 45px;
+  }
+
+  .work-details .work-subtitle {
+    font-size: 18px;
+    margin-bottom: 50px;
+  }
+
+  .work-details .work-role {
+    font-size: 18px;
+  }
+
+  .work-details p {
+    margin-bottom: 25px;
+  }
+
+  .about-myself-wrapper {
+    width: 100%;
+    display: flex;
+    flex-direction: row;
+    align-items: baseline;
+    border-top-right-radius: 13rem;
+    background-color: #fff;
+    padding-bottom: 100px;
+  }
+
+  .about-myself {
+    width: 50%;
+    display: flex;
+    flex-direction: column;
+    margin-top: 10%;
+    padding-left: 6%;
+  }
```

Comment:
- [ ] As you can see in the original design the About myself left section is smaller than the right section. it takes about 40% of the wrapper and the right takes like 60%
![image](https://user-images.githubusercontent.com/43698511/129981805-334db255-e23e-4ac8-8639-42aa2f6c59d6.png)
Yours looks the same size. Kindly fix it
![image](https://user-images.githubusercontent.com/43698511/129982110-1e7858c3-6b8c-44f5-903b-213901eb9c30.png)

Reply:
Oh my bad, I am going to correct that! Thank's for the advice.